client/src/home/a11y-home.yaml

```diff
@@ -29,14 +29,14 @@ home_a11y_subject_section_title:
 home_a11y_non_a11y_redirect_warning:
   transform: [markdown]
   en: |
-    You have been redirect to the text based InfoBase from a page with no text-only equivalent. 
+    You have been redirect to the text-only InfoBase from a page with no text-only equivalent. 
     Pages with no text-only equivalent are purely for visualization purposes and are not the sole presentation of any data in the InfoBase.
     Any information which was presented on the page you were redirected from can be found in the [report builder](#rpb).
   fr: |
-    Vous avez été redirigé vers une version d'InfoBase textuel à partir d'une page sans équivalent textuel.
+    Vous avez été redirigé vers une version d'InfoBase texte seulement à partir d'une page sans équivalent texte seulement.
     Les pages sans équivalent textuel servent à visualiser des donées et ne constituent pas la seule présentation de ces données dans l'InfoBase.
     Toute information présentée sur la page à partir de laquelle vous avez été redirigé se trouve dans le [générateur de rapports](#rpb).
 
 home_a11y_desc:
-  en: Text based Infobase home for improved accessibility
-  fr: Version texte de l’Infobase pour meilleure accessibilité
+  en: Text-only Infobase home, for improved accessibility
+  fr: Version texte seulement de l’Infobase, pour meilleure accessibilité
```

Comment:
Ditto, the French version doesn't really distinguish between "text based" and "text only"

Reply:
I got there from combing through https://www.linguee.com/english-french/translation/text+only.html, which I use when trying to find what the accepted french equivalent for specific common phrases are, but it was pretty inconsistent on this one. Thanks for checking it over!